### PR TITLE
delegis:0.2.0

### PR DESCRIPTION
**Pull request by pklaschka**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: 

#### Features
- Add `#metadata` fields for usage with `typst query`. You can now use `typst query file.typ "<field>" --field value --one` with `<field>` being one of the following to query metadata fields in the command line:
  - `<title>`
  - `<abbreviation>`
  - `<resolution>`
  - `<in-effect>`
- Add `#section[§ 1][ABC]` function to enable previously unsupported special chars (such as `*`) in section headings. Note that this was previously possible using `#unnumbered[§ 1\ ABC]`, but the new function adds a semantically better-fitting alternative to this fix.
- Improve heading style rules. This also fixes an incompatibility with `pandoc`, meaning it's now possible to use `pandoc` to convert delegis documents to HTML, etc.
- Set the footnote numbering to `[1]` to not collide with sentence numbers.
#### Bug Fixes
- Fix a typo in the `str-draft` variable name that lead to draft documents resulting in a syntax error.
- Fix hyphenation issues with the abbreviation on the title page (hyphenation between the parentheses and the abbreviation itself)

### CODE REVIEW

File: packages/preview/delegis/0.2.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#import "@preview/delegis:0.1.0": *
```
Comment (elegaanz) [inline packages/preview/delegis/0.2.0/template/main.typ:1]: The import here is outdated.

File: packages/preview/delegis/0.2.0/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+While the template is designed to be used in German documents, all strings are customizable. You can have a look at the `delegis.typ` to see all available parameters.
+
+## General Usage
+
+While this `README.md` gives you a brief overview of the package's usage, we recommend that you use the template (in the `template` folder) as a starting point instead.
+
+### Importing the Package
+
+```typst
+#import "@preview/delegis:0.1.0": *
```
Comment (elegaanz) [inline packages/preview/delegis/0.2.0/README.md:16]: Same for this import.

File: packages/preview/delegis/0.2.0/template/main.typ [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+#show: it => delegis(
+  // Metadata
+  title: "Vereinsordnung zu ABCDEF",
+  abbrev: "ABCDEFVO",
```
Comment (elegaanz) [inline packages/preview/delegis/0.2.0/template/main.typ:6]: This argument was renamed to `abbreviation` I think.

File: packages/preview/delegis/0.2.0/README.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+### Initializing the template
+
+```typst
+#show: it => delegis(
+  // Metadata
+  title: "Vereinsordnung zu ABCDEF", // title of the law/bylaw/...
+  abbrev: "ABCDEFVO", // abbreviation of the law/bylaw/...
```
Comment (elegaanz) [inline packages/preview/delegis/0.2.0/README.md:25]: It should be renamed here too.

**Comment by pklaschka** [comment]
@elegaanz Thanks for your review! Those were stupid, avoidable issues (as the `abbreviation` one resulted from me fixing it after a review in #420 , then forgetting to also adjust it in the source repo, and now copying the wrong version over again). Sorry about that!

I have addressed your comments. Thanks and again: sorry about that.

**Comment by elegaanz** [comment]
No problem, thanks for fixing that.

**Pull request merged**